modules/sdk-coin-trx/src/lib/utils.ts

```diff
@@ -393,3 +406,43 @@ export function isValidRawTransactionFormat(rawTransaction: any): boolean {
 export function toHex(buffer: Buffer | Uint8Array): string {
   return hex.encode(buffer, true);
 }
+
+/**
+ * Returns a Keccak-256 encoded string of the parameters
+ *
+ * @param types - strings describing the types of the values
+ * @param values - value to encode
+ * @param methodId - the first 4 bytes of the function selector
+ */
+export function encodeDataParams(types: string[], values: any[], methodId: string): string {
+  types.forEach((type, index) => {
+    if (type == 'address') {
+      values[index] = values[index].replace(ADDRESS_PREFIX_REGEX, '0x');
+    }
+  });
+
+  const abiCoder = new AbiCoder();
+  let data;
+  try {
+    data = abiCoder.encode(types, values);
+  } catch (e) {
+    console.log(e); // do something with this error i.e. throw it
```

Comment:
nit: Is there a resolution for this comment?

Reply:
i have another pr for trx tokens coming up, ill add the resolution as a follow up in that, so the transaction builder is merged.